ConceptMap/ConceptMap-Genomics-patient-bornstatus.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType": "ConceptMap",
+  "id": "genomics-patient-bornstatus",
+  "url": "https://fhir.nhs.uk/ConceptMap/genomics-patient-bornstatus",
+  "version": "0.1.0",
+  "name": "GenomicsPatientBornStatus",
+  "title": "Genomics Patient Born Status",
+  "status": "draft",
+  "date": "2024-08-19T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording of concept to indicate if the patient is a fetal. This is intended for the element Patient.extension:fetalStatus",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+ "targetUri": "http://hl7.org/fhir/ValueSet/patient-fetalstatus",
+  "group": [
+    {
+      "target": "http://hl7.org/fhir/patient-bornstatus",
```

Comment:
Note, the codesystem targeted contains the codes born, unborn, fetal demise and unknown (I think we should either use this codesystem as it matches our needs better or switch to the non-deprecated version, in which case this URL should change)